samples/data/dnn/action_classification.txt

```diff
@@ -0,0 +1,400 @@
+abseiling
```

Comment:
Please add the name of dataset to a name of file: `action_recongnition_kinetics.txt`